configs/relays/.my_env_file.env

```diff
@@ -0,0 +1,20 @@
+SUMMA_RELAY_ETHER_HOST=localhost
+
+SUMMA_RELAY_ETHER_PORT=8545
+
+SUMMA_RELAY_OPERATOR_KEY=d6ce6a6ca295bbef9ddee79e583fabdcdd98290bbea2dc03ff8317ccf70d3f87
+
+SUMMA_RELAY_ETH_NETWORK=internal
+
+SUMMA_RELAY_ETH_CHAIN_ID=1101
+
+SUMMA_RELAY_BCOIN_HOST=bcoin.default.svc.cluster.local
+
+# TODO: this should be filled
```

Comment:
We need to include this in README.

Reply:
I just used this secret directly. This testnet bitcoin node belongs to us and is behind a VPN. I think we should worry about their confidentiality.